hrabsence/hrabsence.php

```diff
@@ -454,7 +454,7 @@ function hrabsence_civicrm_tabs(&$tabs, $contactID) {
     )),
     'count' => $absenceDuration/(8*60),
     'title' => ts('Absences'),
-    'weight' => 300
+    'weight' => 10
```

Comment:
Why the weight starts with negative numbers (-200, -190, -180) and them jumps to 10?